test/uk/gov/hmrc/residencenilratebandcalculator/controllers/NoAdditionalThresholdAvailableControllerSpec.scala

```diff
@@ -41,8 +41,9 @@ class NoAdditionalThresholdAvailableControllerSpec
     with WithCommonFakeApplication {
 
   val fakeRequest: FakeRequest[AnyContentAsEmpty.type] = FakeRequest("", "")
-
-  val injector: Injector = fakeApplication.injector
+  val userAnswers                                      = mock[UserAnswers]
+  val reason                                           = mock[Reason]
```

Comment:
Nit: Since these 2 mocks are only used in a single test, you could create them in the test case itself.